### PR TITLE
fix: customClick method is not working #201

### PR DESCRIPTION
**Pull request by marcelovani**
## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Latest master code has been merged into this branch
- [ ] No commented out code (if required, place // TODO above with explanation)
- [ ] No linting issues
- [ ] Build is successful
- [ ] Updated the documentation
- [ ] Added tests to cover changes
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by DalerAsrorov — APPROVED** [review]

**Comment by marcelovani** [comment]
That was quick.
Are you planning to make a new release?

**Comment by marcelovani** [comment]
@DalerAsrorov the last release still doen't show here https://www.npmjs.com/package/britecharts-react

**Comment by Golodhros** [comment]
> @DalerAsrorov the last release still doen't show here https://www.npmjs.com/package/britecharts-react

True, something didn't work on our release process, I will check it out!

**Comment by Golodhros** [comment]
It should be good now!

**Comment by marcelovani** [comment]
I am not sure if I am doing something wrong but the minified version in the dist folder still doesen't containt the customClick method 

How to replicate
- Create a new folder, go into the folder
- run npm i britecharts-react
- run less node_modules/britecharts-react/dist/britecharts-react.min.js
- cat node_modules/britecharts-react/dist/britecharts-react.min.js | grep britechartsCustomEvents

I can only see `britechartsCustomEvents=["customMouseOver","customMouseMove","customMouseOut"]`

How do I do to make the minified js include the fix?

**Comment by Golodhros** [comment]
Oh, my bad, let me try again.

**Comment by Golodhros** [comment]
OK, I tried again, give it some minutes and try the version 0.6.5

**Comment by marcelovani** [comment]
0.6.5 is working now. Thanks
